content/pages/2023/about.md

```diff
@@ -6,32 +6,39 @@ section: euroscipy_2023
 template: page
 slug: about_2023
 ---
+### EuroSciPy Conference
+The EuroSciPy is an annual non-profit European conference on Scientific Python. It is a cross-disciplinary gathering focused on the use and
+development of the Python language in scientific research. This event strives
+to bring together both users and developers of scientific tools, as well as
+academic research and state-of-the-art industry.
```

Comment:
Do you think that we should add the info Python VB in here or it should be in another page?

Reply:
I will add a sentence about Python SV here too.